docs/book/src/quick-start.md

```diff
@@ -190,13 +213,40 @@ Build and push your image to the location specified by `IMG`:
 ```bash
 make docker-build docker-push IMG=<some-registry>/<project-name>:tag
 ```
+If you don’t want to push the image to the mirror repository, just build it
+
+```bash
+make docker-build IMG=<some-registry>/<project-name>:tag
+```
+
+When you build the image, you may get an error similar to `dial tcp x.x.x.x:443: i/o timeout`, You need to edit the `Dockerfile` and set the go proxy
+
+```bash
+#Set the go proxy above "RUN go mod download"
+
+#Go version >=1.13
+RUN go env -w GO111MODULE=on
+RUN go env -w GOPROXY=https://goproxy.io,direct
+RUN go mod download
+
+#Go version <=1.12
+RUN export GO111MODULE="on"
+RUN export GOPROXY="https://goproxy.io"
+RUN go mod download
+
+```
+
+You may get another error similar to `Get https://gcr.io/v2/: net/http: request canceled while waiting for connection (Client.Timeout exceeded while awaiting headers)`, You need to edit the `Dockerfile` and replace the image `gcr.io/distroless/static:nonroot`
 
 Deploy the controller to the cluster with image specified by `IMG`:
 
 ```bash
 make deploy IMG=<some-registry>/<project-name>:tag
 ```
 
+If the pod's status is `ErrImagePull`,You need to edit the file `config/default/manager_auth_proxy_patch.yaml` and replace the image.
```

Comment:
what was the problem faced and what was the edition that you did in this case? Could you please raise an issue for we are able to check? It might be something that could be solved by the tool.

Reply:
All right,It is more appropriate to the issue or FAQ